Tests/Foundation/TestJSONEncoder.swift

```diff
@@ -835,9 +800,9 @@ class TestJSONEncoder : XCTestCase {
                 try decode(type, value)
                 XCTFail("Decode of \(value) to \(type) should not succeed")
             } catch DecodingError.dataCorrupted(let context) {
-                XCTAssertEqual(context.debugDescription, errorMessage)
+                // Pass
```

Comment:
Not really valuable to check the exact message here. Just needs to throw.

Reply:
`XCTAssertThrowsError`?

Reply:
Much better, thanks.